### ISSUE TITLE
Docs: $search->filter() method takes 'in' operator

### ISSUE THREAD

**Issue by milanmadar** [manticoresoftware/manticoresearch-php#63]
Perhaps list the `in` operator [here](https://gitlab.com/manticoresearch/manticoresearch-php/uploads/89beab258c8804c347c25e17b1509037/1634715401_searchclass.md_filter-and-notfilter), used for MVA attributes (took me some time and failed tests to dig it up).

**Comment by tchartron** [comment]
I agree with this `in`operator should be added in the docs of filter method as seen here : https://github.com/manticoresoftware/manticoresearch-php/blob/1ccb8374320579ab372e32f2b88abb5bb5562560/src/Manticoresearch/Search.php#L165

**Comment by githubmanticore** [comment]
➤ Nick Sergeev commented:
 

 
Done in https://github.com/manticoresoftware/manticoresearch-php/pull/77

**Comment by marclaporte** [comment]
Is there anything remaining to do here?

**Comment by sanikolaev** [comment]
> Is there anything remaining to do here?

No. Closed.
